app/controllers/pxe_controller/iso_datastores.rb

```diff
@@ -114,7 +114,7 @@ def iso_datastore_get_form_vars
   # Set form variables for edit
   def iso_datastore_set_form_vars
     @edit = {}
-    @edit[:emses] = ManageIQ::Providers::Redhat::InfraManager.without_iso_datastores.order(:name)
+    @edit[:emses] = ExtManagementSystem.supports?(:create_iso_datastore)
```

Comment:
Addressing https://github.com/ManageIQ/manageiq-ui-classic/pull/8116/files#r806082193

Reply:
I don't think this is right.  I think this should be `ExtManagementSystem.supporting(:create_iso_datastore)`?

Additionally, I don't think we want to lose the `.order(:name)`

Reply:
@Fryguy Thanks for the catch 👍

Reply:
Thanks guys 👍